tests/basic/pool_function_test.go

```diff
@@ -504,3 +505,112 @@ var _ = Describe("{PoolExpandWhileResizeDiskInProgress}", func() {
 	})
 
 })
+
+var _ = Describe("{PoolExpandResizeClusterNoQuorum}", func() {
+	//1) Deploy px with cloud drive.
+	//2) Make Cluster out of quorum
+	//3) Expand a healthy pools by resize-disk
+
+	var testrailID = 34542845
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/34542845
```

Comment:
For some reason, this testrail link is not opening actual testrail testcase

Reply:
done